v2x/yosys/utils.py

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env python3
 import re
-"""The JSON Yosys outputs isn't acutally compliant JSON, as it contains C-style
-comments. These must be stripped."""
+
+CLOCK_NAME_REGEX = re.compile(r"[a-z_]*clk[a-z0-9]*$")
 
 
 def strip_yosys_json(text):
+    """The JSON Yosys outputs isn't acutally compliant JSON, as it contains C-style
+    comments. These must be stripped."""
     stripped = re.sub(r'\\\n', '', text)
     stripped = re.sub(r'//.*\n', '\n', stripped)
     stripped = re.sub(r'/\*.*\*/', '', stripped)
     return stripped
+
+
+def is_clock_name(name):
+    """
+    Returns true if the port name correspond to a clock according to arbitrary
+    regular expressions.
```

Comment:
Please add a couple of quick doctests for the clock name detection.

Reply:
Added simple doctests.